bee-common/bee-common-derive/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use proc_macro::{self, TokenStream};
+use proc_macro_error::{abort, proc_macro_error};
+use quote::{quote, ToTokens};
+use syn::{parse_macro_input, Data, DeriveInput, Type};
+
+mod packable;
```

Comment:
Alright I'm being too picky now, but for consistency with the rest of the codebase, can we put `mod`s before `use`s please ?

Reply:
there is no rest of the codebase :trollface: (JK)